### DISCUSSION TITLE
cannot import name 'SentenceSegmenter' from 'spacy.pipeline'

### DISCUSSION THREAD

**Discussion by rohan2734** [explosion/spaCy#7022] [Help: Coding & Implementations] [answered]
## cannot import name 'SentenceSegmenter' from 'spacy.pipeline'

this is the code
```
from spacy.pipeline import SentenceSegmenter
```
this is the error
```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-538101332913> in <module>
----> 1 from spacy.pipeline import SentenceSegmenter

ImportError: cannot import name 'SentenceSegmenter' from 'spacy.pipeline' (C:\Users\Hp\anaconda3\envs\nplenv\lib\site-packages\spacy\pipeline\__init__.py)
```
## Your Environment

* Operating System:windows
* Python Version Used:3.8
* spaCy Version Used:(spaCy v3.0.1)
* Environment Information: conda

**Comment by adrianeboyd** [comment, marked as answer]
Hi, `SentenceSegmenter` has been removed in v3, see: https://spacy.io/usage/v3#incompat-removed

**Comment by rohan2734** [comment]
Thankyou, for mentioning it, I was searching a lot for it, and finally you have answered it. I will check its replacement.
